sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -171,21 +187,9 @@ object JdbcUtils extends Logging {
       val name = field.name
       val typ: String =
         dialect.getJDBCType(field.dataType).map(_.databaseTypeDefinition).getOrElse(
-          field.dataType match {
```

Comment:
Why these entries move to `JdbcDialects#getCommonJDBCType`?
ISTM common types among databases,  i.e., SQL standards ones, are put in `JavaUtils`.